agent/util-scripts/pbench-tool-meister-start

```diff
@@ -342,6 +342,12 @@ def main(argv):
         ), f"bad channel: {resp!r}"
         assert resp["data"] == 1, f"bad data: {resp!r}"
 
+    # 2.5. Add tool metadata json to redis
+    #FIXME: ABSOLUTE PATH IS BAD (also add try/catch)
+    with open('/opt/pbench-agent/tool-scripts/meta.json') as json_file:
```

Comment:
We can add the path in the pbench-agent config file I guess

Reply:
We know the path, we just need the installation directory, to that access the path relative to that:

```
with Path(config.install_dir, "tool-scripts", "meta.json").open("r") as json_file:
    ...
```